spec/features/jobs/user_deletes_a_job_spec.rb

```diff
@@ -0,0 +1,17 @@
+require 'rails_helper'
+
+describe 'User deletes a job' do
+  scenario 'a user can delete a job' do
+    company = Company.create(name: "Apple")
+    job = company.jobs.create(title: 'Web Developer',
+                              level_of_interest: 8,
+                              city: 'Denver',
+                              description: 'Awesome Stuff')
```

Comment:
Could potentially expect that `company.jobs.count` is one here, and then after the delete expect that it's zero.